translations/README/SWAHILI.md

```diff
@@ -7,8 +7,8 @@
 [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)][twit]
 
 [![Discord](https://badgen.net/discord/online-members/tWkvS4ueVF?label=Jiunge%20Our%20Discord%20Server&icon=discord)](https://discord.gg/tWkvS4ueVF 'Jiunge na seva yetu ya Discord !')
-[![PRs Karibu](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://syknapse.github.io/Contribute-To-This -Mradi/)
-[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://syknapse.github.io/Changia-Kwa-Hii- Mradi/)
+[![PRs Karibu](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://syknapse.github.io/Contribute-To-This-Mradi/)
+[![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://syknapse.github.io/Changia-Kwa-Hii-Mradi/)
```

Comment:
the links shouldn't be translated. kindly reverse this and change it to same as original README file (english)